deepmd/descriptor/se_r.py

```diff
@@ -392,15 +392,15 @@ def _pass_filter(self,
                                      [ 0, start_index*      self.ndescrpt],
                                      [-1, natoms[2+type_i]* self.ndescrpt] )
                 inputs_i = tf.reshape(inputs_i, [-1, self.ndescrpt])
-                layer = self._filter_r(tf.cast(inputs_i, self.filter_precision), type_i, name='filter_type_'+str(type_i)+suffix, natoms=natoms, reuse=reuse, trainable = trainable, activation_fn = self.filter_activation_fn)
+                layer = self._filter_r(self.filter_precision, type_i, name='filter_type_'+str(type_i)+suffix, natoms=natoms, reuse=reuse, trainable = trainable, activation_fn = self.filter_activation_fn)
```

Comment:
The first variable should be "inputs_i" instead of “self.filter_precision”. It couldn't pass UT but it was merged into devel.

Reply:
In fact, this line is not covered by UT.

Reply:
Yes, it's covered by source/tests/test_model_compression_se_r.py, a UT just added in PR #1361.